.circleci/config.yml

```diff
@@ -1,42 +1,118 @@
 version: 2.1
 
-jobs:
-  standards:
+commands:
+  install-dependencies:
+    description: "Install development dependencies."
+    steps:
+      - run: composer install
+      - run: npm ci
+  mkdir_artifacts:
+    description: "Make Artifacts directory"
+    steps:
+      - run:
+          command: |
+            [ ! -d "/tmp/artifacts" ] && mkdir /tmp/artifacts &>/dev/null
+
+executors:
+  base:
+    docker:
+      - image: circleci/buildpack-deps:latest
+    working_directory: /tmp
+  php_node:
     docker:
       - image: circleci/php:7.3.3-stretch-node-browsers
+    working_directory: /tmp/genesis-sample
+  python:
+    docker:
+      - image: circleci/python:3.7-stretch
+    working_directory: /tmp
+
+jobs:
+  checkout:
+    executor: base
     steps:
-      - checkout
+      - checkout:
+          path: genesis-sample
+      - persist_to_workspace:
+          root: /tmp
+          paths:
+            - genesis-sample
+
+  standards:
+    executor: php_node
+    steps:
+      - attach_workspace:
+          at: /tmp
       - install-dependencies
       - run: composer phpcs
       - run: npm run lint:css
       - run: npm run lint:js
 
   bundle:
-    docker:
-      - image: circleci/node:12.12.0
+    executor: php_node
     steps:
-      - checkout
+      - attach_workspace:
+          at: /tmp
       - run: npm ci
       - run: npm run zip
-      - run: mkdir /tmp/artifacts
+      - mkdir_artifacts
       - run: mv ./*.zip /tmp/artifacts/.
+      - persist_to_workspace:
+          root: /tmp
+          paths:
+            - artifacts
+            - genesis-sample
       - store_artifacts:
           path: /tmp/artifacts/
 
-commands:
-  install-dependencies:
-    description: "Install development dependencies."
+  create_data_file:
+    executor: php_node
     steps:
-      - run: composer install
-      - run: npm ci
+      - attach_workspace:
+          at: /tmp
+      - run: ls -lash
+
+  deploy_s3:
+    executor: python
+    steps:
+      - attach_workspace:
+          at: /tmp
+      - aws-s3/sync:
+          from: "/tmp/artifacts/"
+          to: "s3://<need_bucket_url>"
```

Comment:
@mattlawck are we gonna have another s3 bucket for genesis stuff or use one bucket for all stuff coming out of the info service?

Reply:
@markkelnar my opinion would be a single bucket with folders , since it is one API for these different things. I can't think of an advantage for having many s3 buckets that isn't covered by the use of folders. 

As always, open to any and all considerations as well as my opinion not being used 😄

Reply:
I agree with this @mattlawck .  Do we care that the current name of the bucket being used is atomic-block specific? update.atomicblocks.com

Reply:
Nope.

Reply:
Noted for the next PR I'll be making